prow/config/config.go

```diff
@@ -70,6 +70,8 @@ const (
 	// it up if present. This allows components to include the config version
 	// in their logs, which can be useful for debugging.
 	ConfigVersionFileName = "VERSION"
+
+	DefaultTenantID = "Default"
```

Comment:
nit: can we naming this a bit longer so that there is no chance someone can accidentally come up the same name in their prow config?

Reply:
Done